app/sidekiq/worker_job.rb

```diff
@@ -0,0 +1,19 @@
+class WorkerJob
+  include Sidekiq::Job
+
+  def perform
+    yesterday_rank = Rank.create(date: Date.yesterday)
+
+    books = Book.all
+    books.each do |book|
+      view = Rails.cache.fetch("views/#{book.id}") { 0 }
+      BookRank.create(book: book, rank: yesterday_rank, view: view)
+    end
+
+    Rails.cache.clear
```

Comment:
Be careful using `Rails.cache.clear` here, since it flashes all redis data, so the sidekiq queues and many other staffs that we stored in redis will be deleted.